MODULE.bazel

```diff
@@ -16,3 +16,85 @@
 #
 # For more details, please check https://github.com/bazelbuild/bazel/issues/18958
 ###############################################################################
+module(
+    name = "iceoryx2",
+    version = "0.1.0",
+)
+
+bazel_dep(name = "bazel_features", version = "1.30.0")
+bazel_dep(name = "bazel_skylib", version = "1.7.1")
+
+bazel_dep(name = "googletest", version = "1.15.2", dev_dependency = True)
+
+bazel_dep(name = "platforms", version = "0.0.11")
+bazel_dep(name = "rules_cc", version = "0.1.1")
+bazel_dep(name = "rules_rust", version = "0.58.0")
+bazel_dep(name = "iceoryx", version = "2.95.7")
+git_override(
+    module_name = "iceoryx",
+    commit = "6760c0088c61e8bf68070e2d163a37caab7515b1",
+    remote = "https://github.com/qorix-group/iceoryx.git",
+)
```

Comment:
We need to change this to upstream once https://github.com/eclipse-iceoryx/iceoryx/pull/2505 got merged

Reply:
Once #301 is completed, likely in the next two weeks, this is superfluous and can be removed.